try-Catch
try:
   a = 20
   b = 0
   print(a / b)
except ZeroDivisionError:
   print("There is divide by zero error")

O/p-There is divide by zero error

try-Catch-finally
try:
   a = 20
   b = 0
   print(a / b)
except ZeroDivisionError:
   print("There is divide by zero error")
finally:
   print("This is going to execute always")

O/p-There is divide by zero error
This is going to execute always


